bigbluebutton-config/bin/bbb-conf

```diff
@@ -161,6 +161,7 @@ HTML5_CONFIG_OLD=/usr/share/meteor/bundle/programs/server/assets/app/config/sett
 HTML5_CONFIG=/usr/share/meteor/bundle/programs/server/assets/app/config/settings.yml
 
 KURENTO_CONFIG=/usr/local/bigbluebutton/bbb-webrtc-sfu/config/default.yml
+KURENTO_ETC_CONFIG=/etc/bigbluebutton/bbb-webrtc-sfu/production.yml
```

Comment:
You can use yq to merge and override entries with precedence of $KURENTO_ETC_CONFIG.
See https://mikefarah.gitbook.io/yq/v/v2.x/merge for details.

```suggestion
KURENTO_DEFAULT_CONFIG=/usr/local/bigbluebutton/bbb-webrtc-sfu/config/default.yml
KURENTO_ETC_CONFIG=/etc/bigbluebutton/bbb-webrtc-sfu/production.yml
KURENTO_CONFIG=$(yq m -x $KURENTO_DEFAULT_CONFIG $KURENTO_ETC_CONFIG)
```

Now, you must change all snippets where $KURENTO_CONFIG occurs, so that it would not consider $KURENTO_CONFIG as a file!
Like so:
`KURENTO_SIP=$(echo "$KURENTO_CONFIG" | yq r - freeswitch.sip_ip)`